docs/modules/clients/pages/java.adoc

```diff
@@ -1074,27 +944,82 @@ YAML::
 ----
 hazelcast-client:
   network:
+    cluster-members:
+      - 127.0.0.1
+      - 127.0.0.2
+    outbound-ports:
+      - 34600
+      - 34700-34710
+    cluster-routing:
+     mode: ALL_MEMBERS
     redo-operation: true
+    connection-timeout: 60000
+    socket-options:
+      ...
+    socket-interceptor:
+      ...
+    ssl:
+      enabled: false
+      ...
+    aws:
+      enabled: true
+      connection-timeout-seconds: 11
+      ...
+    gcp:
+      enabled: false
+      ...
+    azure:
+      enabled: false
+      ...
+    kubernetes:
+      enabled: false
+      ...
+    eureka:
+      enabled: false
+      ...
+    icmp-ping:
+      enabled: false
+      ...
+    hazelcast-cloud:
+      enabled: false
+      discovery-token: EXAMPLE_TOKEN
+    discovery-strategies:
+      node-filter:
+        class: DummyFilterClass
+      discovery-strategies:
+        - class: DummyDiscoveryStrategy1
+          enabled: true
+          properties:
+            key-string: foo
+            key-int: 123
+            key-boolean: true
 ----
 ====
 
-Programmatic Configuration:
+==== Programmatic configuration
+
+The following example programmatic `network` configuration includes all the parent configuration attributes:
+// is attributes the right term here?
```

Comment:
sorry, I didnt get what this sentence means by "parent configuration".

Reply:
I got it, thanks. For programmatic configuration, I would not use "attribute". Maybe we should use all the main setter methods? I'm not sure.

Reply:
I like the second suggestion, thanks - will update